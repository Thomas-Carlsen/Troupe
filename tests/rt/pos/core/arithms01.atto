let val x = 10
    val y = x * 10 + 2
    val z = 10 + (if true then 11 else 22)
    val w = if let val x = true in x end then (fn t => true) 0 else false
    fun f x y = x + y * 10
in if x >= 10 then
    f (x - (f (y - 2) 2) ) ( 2 - 2 * 3)
   else
    120
end
