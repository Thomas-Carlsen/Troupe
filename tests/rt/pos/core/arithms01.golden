2019-12-09T13:24:53.968Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: -150@{}%{}
